quantmsrescore/ms2_model_manager.py

```
@@ -0,0 +1,173 @@
+from peptdeep.model.rt import AlphaRTModel
+from peptdeep.model.ccs import AlphaCCSModel
+from peptdeep.model.charge import ChargeModelForModAASeq
+import os
+from peptdeep.utils import logging
+
+
+class MS2ModelManager(ModelManager):
+    def __init__(self,
+                 mask_modloss: bool = False,
+                 device: str = "gpu",
+                 model_dir: str = None,
```

Ruff (0.14.7)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)